man/kernel-install.xml

```diff
@@ -34,9 +34,13 @@
 
   <refsect1>
     <title>Description</title>
-    <para><command>kernel-install</command> is used to install and remove kernel and initramfs images to and
-    from the boot loader partition, referred to as <varname>$BOOT</varname> here. It will usually be one of
-    <filename>/boot/</filename>, <filename>/efi/</filename>, or <filename>/boot/efi/</filename>, see below.
+    <para><command>kernel-install</command> is used to install and remove kernel and initrd images
+    <footnote><para>Nowadays actually "initramfs" rather than "initrd". See
```

Comment:
initramfs is also wrong, as it's not ramfs but tmpfs anymore.

Reply:
maybe just say "initial tmpfs"?

Reply:
I checked the kernel code, and ramfs is still a possibility. I adjusted the text to be less roundabout and say "CPIO archive" and "tmpfs or ramfs", just to make this all easier to grok.

Reply:
Please drop the refernce to ramfs. That's historical. They kept it for compat, but noone does that.